documentation/admin.py

```diff
@@ -42,9 +42,9 @@ class IDEAdmin(PageAdmin, VersionAdmin):
         }),
     )
 
-    # Override the "view on site" button to use local /documentation/ instead of docs.code.org path
+    # Override the "view on site" button to use local /docs/ instead of slug which does not have /docs
     def view_on_site(self, obj):
-        return '/documentation%s' % obj.get_absolute_url()
+        return '/docs%s' % obj.get_absolute_url()
```

Comment:
It's a bit misleading that the method called `get_absolute_url` is not returning an absolute url. It seems like we still need that method though, because we are using its return value as a suffix after /docs/ as well as /documentation/ . Could you rename this to `get_url_suffix` or `get_url_part` or something like that?

If you are planning to change the rest of the urls over to use /docs instead of /documentation, then another idea would be to wait until that is done and then have `get_absolute_url` actually return an absolute url, including /docs/ prefix.

Reply:
oops, just realized this is overriding a django method: https://docs.djangoproject.com/en/1.8/ref/models/instances/#get-absolute-url

I'm not sure when this method is getting used by django behind the scenes, but it makes me think that it's important to define it accurately, so that it produces a working url, including /docs. then you can change the implementations of the methods which append this to '/documentation' to build their url a different way.

Reply:
I think I'm going to hold on this for now until we figure out docs.code.org - then we can just change the slug for everything over to /docs/..... and do the clean up work then. I will add it to my list of things to follow up on in jira